server/routes/index.ts

```diff
@@ -289,6 +290,10 @@ export default function routes(
   router.use(adjudicationUrls.punishmentComment.root, PunishmentCommentRoutes({ userService, punishmentsService }))
   router.use(adjudicationUrls.punishment.root, PunishmentRoutes({ userService, punishmentsService }))
   router.use(adjudicationUrls.punishmentSchedule.root, PunishmentScheduleRoutes({ userService, punishmentsService }))
+  router.use(
```

Comment:
Could it be worth putting this behind the feature flag we added with https://github.com/ministryofjustice/hmpps-manage-adjudications/pull/695?